src/Behat/Behat/Output/Node/EventListener/JUnit/JUnitDurationListener.php

```diff
@@ -75,11 +75,9 @@ private function captureAfterScenarioEvent(Event $event): void
 
         $key = $this->getHash($event->getScenario());
         $timer = $this->scenarioTimerStore[$key];
-        if ($timer instanceof Timer) {
-            $timer->stop();
-            $this->resultStore[$key] = $timer->getTime();
-            unset($this->scenarioTimerStore[$key]);
-        }
+        $timer->stop();
```

Comment:
$timer is always a Timer

Reply:
is it guaranteed that it is a timer, or could it be `null` with a notice about the key not being defined ?

Reply:
Ah, yes, it could be that it is not defined. I implemented another fix